R/adverb-safely.R

```diff
@@ -10,7 +10,11 @@
 #' function (a verb). If you'd like to include a function created an adverb
 #' in a package, be sure to read [faq-adverbs-export].
 #'
-#' @inheritParams map
+#' @param .f A function to modify, specified in one of the following ways:
+#'   * A named function, e.g. `mean`.
+#'   * An anonymous function, e.g. `function(x) x + 1`.
+#'   * A lambda function, e.g. `\(a) a + 1`.
+#'   * A formula, e.g. `~ .x + 1`. Only appropriate for simple tasks.
```

Comment:
Are we superseding formulas in purrr 1.0? If yes, probably best to omit mentioning them prominently in doc.

Reply:
Though I guess that requires first solving the issue of using base lambdas in example. As long as we use formulas in examples, they need to be documented.

Reply:
Yeah, that was exactly my thought process 😄